docs/user-guide/options.md

```diff
@@ -36,7 +36,13 @@ CLI flag: `--fix`
 
 Automatically fix, where possible, problems reported by rules.
 
-For CSS with standard syntax, Stylelint uses [postcss-safe-parser](https://www.npmjs.com/package/postcss-safe-parser) to fix syntax errors.
+Options are:
+
+- `"lax"` (default) - uses [postcss-safe-parser](https://www.npmjs.com/package/postcss-safe-parser) to fix as much as possible, even when there are syntax errors
+- `"strict"` - uses [PostCSS Parser](https://postcss.org/api/#postcss-parser) and only fixes problems when there are no syntax errors
+
+> [!TIP]
+> If you don't care about strictness now, we recommend using `--fix` because `"strict"` may become the default in the future.
```

Comment:
```suggestion
> If you don't care about strictness now, we recommend using the boolean flag `--fix`—which _currently_ maps to `--fix=lax`—without any value because `"strict"` may become the new default in the future.
```

Reply:
Thank you both for your suggestions. 
Since it says `"lax" (default)`, I thought it’s unnecessary to mention that it’s mapped!